tests/test_helpers_fsutil.py

```diff
@@ -1,10 +1,175 @@
 import os
+from unittest.mock import Mock
 
 import pytest
 
+from dissect.target.filesystem import VirtualFilesystem
 from dissect.target.helpers import fsutil
 
 
+@pytest.mark.parametrize(
```

Comment:
Would it be possible to create a unit test where the target is a regular Windows target, but it has a "Unix-y" filesystem? E.g. an EXTFS partition.

Reply:
What would that accomplish wrt extra tests done?

At the moment support for this is not great, you won't be able to get to files which are valid on Unix systems, differ only in case or have a `\` in their name, but not on Windows, as there is no extfs-on-windows shim which translates these names.

Note that case sensitivity and alt path separators for all filesystems in a target are forced through its `RootFilesystem`. These properties are set by `OSPlugin.create()` which is called either by a loader or by `Target`'s `_init_os()`

Reply:
Mostly a yes/no if this will work or not, but if I understand you correctly, this won't work right now?

Reply:
It will work but with limitations and unknown failure modes.

Reply:
Can you create a ticket to see if this is possible to properly support?

Reply:
Done